indico/modules/events/editing/schemas.py

```diff
@@ -139,6 +140,28 @@ class Meta:
     review_conditions_valid = fields.Boolean()
 
 
+class EditableBasicSchema(mm.ModelSchema):
+    class Meta:
+        model = Editable
+        fields = ('id', 'type', 'state', 'editor')
+
+    state = fields.String()
+    editor = fields.Nested(UserSchema, only=('id', 'avatar_bg_color', 'full_name'))
+
+
+class EditingEditableListSchema(mm.ModelSchema):
+    class Meta:
+        model = Contribution
+        fields = ('id', 'friendly_id', 'title', 'code', 'editable')
+
+    editable = fields.Method('_get_editable')
+
+    def _get_editable(self, contribution):
+        editable_type = self.context['editable_type']
+        editable = {editable for editable in contribution.editables if editable.type == editable_type}
+        return EditableBasicSchema().dump(editable)
```

Comment:
This will silently error, since you are passing a `set` to the `dump` method which expects a single object.

Reply:
What should be the correct way to do it since editable will always be a single object?

Reply:
add a `get_editable(self, editable_type)` method to `Contribution` which contains e.g. this:

```python
return next((e for e in self.editables if e.type == editable_type), None)
```